packages/components/src/feature_alert_label/index.module.scss

```diff
@@ -0,0 +1,40 @@
+@import 'nebenan-ui-kit';
+
+.label {
+  display: inline-flex;
+
+  &.is-position-right .badge {
+    margin-left: 10px;
+  }
+
+  &.is-position-left .badge {
+    margin-right: 10px;
+    order: -1;
+  }
+
+  &.is-position-top,
+  &.is-position-bottom {
+    flex-direction: column;
+    align-items: flex-start;
+
+    .badge {
+      flex-grow: 0;
+    }
+  }
+
+  &.is-position-bottom .badge {
+    margin-top: 5px;
+  }
+
+  &.is-position-top .badge {
+    margin-bottom: 5px;
+    order: -1;
+  }
+}
+
+.badge {
+  color: $color-gray-09;
+  background: linear-gradient(110deg, #33a6b4, $color-blue-01);
```

Comment:
don't we have #33a6b4 in ui-kit?

Reply:
Nope, didn't find any

Reply:
choose nearest color?

Reply:
The nearest one is `$color-blue-01`
So we can make it ` background: linear-gradient(110deg, $color-blue-01,  $color-blue-00);` instead, which will make it a bit darker